themes/getting-started/theme-organization.rst

```diff
@@ -5,7 +5,18 @@ Theme organization
 Directory structure
 ========================
 
-// TODO Needs more details!
+A PrestaShop theme is a set of files which you can edit in order to change the look of your online shop.
+
+Here are a few important tidbits:
+
+- All themes have their files located in the /themes folder, at the root of PrestaShop's folder.
+- Each theme has its own sub-folder, in the main themes folder.
+- Each theme is made of template files (.tpl), image files (.gif, .jpg, .png), one or more CSS files (.css), and sometimes even JavaScript files (.js).
+- Each theme has a preview.png image file in its folder, enabling the shop-owner to see what the theme looks like directly from the back office, and select the theme appropriately.
+
```

Comment:
The theme.yml file has its own page: https://github.com/PrestaShop/docs/blob/master/themes/getting-started/theme-yml.rst
The config folder is explained later in this page, but indeed, it could use more info.